docs/UserGuide.adoc

```diff
@@ -82,49 +82,37 @@ Format: `clear`
 
 ==== Importing eateries: `import`
 
-*Coming in v1.4*
-
 Imports eateries from a text file. +
 Format: `import \f [filename]`
 
 Examples:
 
 * `import \f /home/user/eatme-backup-john.txt`
```

Comment:
I think you can ask Kai Qi about how we're implementing import feature now as I think it's changed quite a bit? But it's up to you la bc I think Kai Qi can always edit this part himself later

Reply:
I'll ask him to check this haha